docs/README.md

```diff
@@ -519,12 +524,47 @@ Renders the customer information. Must be placed inside an [`op-order`](#op-orde
 | -------------------------------------------------- |
 | ![op-order-payment](https://user-images.githubusercontent.com/17712401/89225037-eec36f80-d5af-11ea-82af-267cb69c7d7b.png) |
 
+### `op-order-billing`
+
+Renders the billing details set in CustomData during checkout (must have the following fields: `name`, `phone_number`, `city`, `street` defined ). Must be placed inside an [`op-order`](#op-order) block.
```

Comment:
Custom data is not the same across stores, this should be a custom component instead.

Reply:
Now anyone can create React components in apps:
https://developers.vtex.com/vtex-developer-docs/changelog/react-builder-in-vtex-io-now-in-open-beta

Reply:
Removed everything related billing from order-placed app and moved it to a custom app instead.